### PR TITLE
feat(editors/substation/transformerwinding): add remove button 

### PR DESCRIPTION
**Pull request by marcvanraalte**
Changed in transformer-winding-editor-wizard-edititing.test.ts the argument of querySelector to handle the element with name some.

### CODE REVIEW

**Comment by marcvanraalte** [comment]
This close VS-872

**Comment by marcvanraalte** [comment]
closes #968

File: test/integration/editors/substation/transformer-winding-editor-wizard-editing.test.ts [hunk 1]
```diff
@@ -29,7 +29,9 @@ describe('transformer-winding-editor wizarding editing integration', () => {
         await fixture(
           html`<mock-wizard-editor
             ><transformer-winding-editor
-              .element=${doc.querySelector('TransformerWinding')}
+              .element=${doc.querySelector(
```
Comment (JakobVogelsang) [inline test/integration/editors/substation/transformer-winding-editor-wizard-editing.test.ts:32]: Why is this change on the query necessary?

**Review by JakobVogelsang — COMMENTED** [review]
Most of it LGTM. I cannot understand however what the changed test is for :)

**Review by JakobVogelsang — APPROVED** [review]
LGTM

**Pull request merged**